example/README.md

```diff
@@ -0,0 +1,64 @@
+In this folder, we can see four main packages in different sub-folders, register_certificate, register_contract, execute_contract and validate_ledger. They demonstrate how to use Scalar DL Go Client SDK to send basic operation requests to Scalar DL networks.
```

Comment:
This is a very minor nitpick, but I wonder if it would be easier to read to nest those listed sub-folders in a bulleted list.

like 
```
In this folder, we can see four main packages in different sub-folders:
 - register_certificate
 - ...
 ```

Reply:
Thanks!
Let me revise it.